docs/source/reference/services.md

```diff
@@ -201,8 +201,9 @@ below.
 The reference, or base, implementation is the [`HubAuth`][HubAuth] class,
 which implements the requests to the Hub.
 
-To use HubAuth, you must set the `.api_token`, either programmatically when constructing the class,
-or via the `JUPYTERHUB_API_TOKEN` environment variable.
+To use HubAuth, you must set the `.api_token` instance variable. This can be
+done either programmatically when constructing the class, or via the
+`JUPYTERHUB_API_TOKEN` environment variable. (TODO: Where is this API TOKen set?)
```

Comment:
https://github.com/jupyterhub/jupyterhub/blob/071e375d5fd038e70c29c619d199bf2c9c090d5c/docs/source/reference/rest.md#add-api-tokens-to-the-config-file

Reply:
The examples directory in the repo's root directory has configuration examples for different cases.

Reply:
Is there a way to crosslink to that markdown heading?